### PR TITLE
add offset field tests

### PR DESCRIPTION
**Pull request by Sild**

### CODE REVIEW

File: src/test/java/com/axibase/tsd/api/method/property/PropertyQueryTest.java [hunk 1]
```diff
@@ -1409,6 +1409,138 @@ public void testLimitNegative() throws Exception {
 
     }
```
Comment (KutuzovGregory) [inline src/test/java/com/axibase/tsd/api/method/property/PropertyQueryTest.java]: Why not use the same data for all 3 tests? Just put 3 property and than query them with different offsets. I think that this is the case of parametrised test.

**Comment by KutuzovGregory** [comment]
Why not use the same data for all 3 tests? Just put 3 property and than query them with different offsets. I think that this is the case of parametrised test.

**Comment by Sild** [comment]
Because the tests should be atomic and there is no any reason to use parameterized tests for few similar tests

**Comment by Sild** [comment]
Because the tests should be atomic and there is no any reason to use parameterized tests for few similar tests

**Comment by KutuzovGregory** [comment]
ok

**Comment by rmakulov** [comment]
Ok

**Pull request merged**